data/src/main/java/sample/kanda/data/local/AppDataBase.kt

```diff
@@ -0,0 +1,68 @@
+package sample.kanda.data.local
+
+import android.arch.persistence.db.SupportSQLiteDatabase
+import android.arch.persistence.room.Database
+import android.arch.persistence.room.Room
+import android.arch.persistence.room.RoomDatabase
+import android.content.Context
+import sample.kanda.data.local.contact.ContactDao
+import sample.kanda.data.local.contact.ContactEntity
+import sample.kanda.data.local.contact.LabelDao
+import sample.kanda.data.local.contact.LabelEntity
+import java.util.concurrent.Executors
+
+/**
+ * Created by caique on 2/16/18.
+ */
+@Database(
+        entities = [(ContactEntity::class), (LabelEntity::class)],
+        version = 1,
+        exportSchema = false)
+
+abstract class AppDataBase : RoomDatabase() {
+    abstract fun contactDao(): ContactDao
+    abstract fun labelDao(): LabelDao
+
+
+    companion object {
+        //https://gist.github.com/florina-muntenescu/697e543652b03d3d2a06703f5d6b44b5
```

Comment:
Esse link de Gist dá Not found, e eu não consigo saber a partir dele o pq do `@Volatile`